app/Http/Controllers/PhotoEditorController.php

```diff
@@ -45,13 +49,29 @@ public function rotate(Request $request)
 			return false;
 		}
 
+		// direction is valid?
+		if (($direction != 1) && ($direction != -1)) {
+			Logs::error(__METHOD__, __LINE__, 'Direction must be 1 or -1');
+
+			return false;
```

Comment:
false needs to be a string.
```suggestion
			return 'false';
```